### DISCUSSION TITLE
rules

### DISCUSSION THREAD

**Discussion by mpacker99** [afterxleep/Bouncer#78] [Ideas and Feature Requests]
Anyway of getting a set of rules we can download/import?   surely we can get some sample credit cards, store rules to automatically file messages.

even if the rules are just available here

**Comment by Mr-McMuffin** [comment]
Yes there should be a standard blocking list signature hound does this currently and is completely free; without that it's hard to recommend the app.

**Comment by afterxleep** [comment]
I have thought about this several times, but given the app is used worldwide, that list will be enormous and very hard to create/maintain.   Perhaps I can add some sort of functionality in the future, where anyone can share their filters to a crowdsourced list..  Just an idea though..
